app/containers/Login/index.js

```diff
@@ -13,24 +13,77 @@ import {
     Button, 
     Text
 } from 'native-base';
-import { StyleSheet } from 'react-native';
+import { StyleSheet, Alert, AsyncStorage } from 'react-native';
+import { Actions } from 'react-native-router-flux';
 
 export default class Login extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { 
+            username: '',
+            password: '',
+            loggedIn: false 
+        };
+    }
+
+    onLogin = (usernameInput, passwordInput) => {
+        fetch('http://private-f30431-devsummit.apiary-mock.com/auth/login', {
+            method: 'POST',
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                username: usernameInput,
+                password: passwordInput
+            })
+        }).then((response) => response.json())
+        .then(async (responseJson) => {
+            if (responseJson.message === 'user logged in succesfully') {
```

Comment:
there is actually a `success` variable that decide wether the log in fail or success, use that instead of comparing `message` string